CMakeLists.txt

```
@@ -41,15 +41,44 @@ FIND_PACKAGE( DD4hep REQUIRED )
 FIND_PACKAGE( ROOT REQUIRED )
 FIND_PACKAGE( MarlinTrk REQUIRED )
 
+OPTION( USE_SDHCALCONTENT "Use SDHCALContent" OFF )
+OPTION( USE_APRILCONTENT "Use APRILContent" OFF )
 
-FOREACH( pkg PandoraSDK Marlin MarlinUtil  LCContent DD4hep ROOT MarlinTrk )
+FOREACH( pkg PandoraSDK Marlin MarlinUtil LCContent DD4hep ROOT MarlinTrk )
     IF( ${pkg}_FOUND )
         INCLUDE_DIRECTORIES( SYSTEM ${${pkg}_INCLUDE_DIRS} )
         LINK_LIBRARIES( ${${pkg}_LIBRARIES} )
         ADD_DEFINITIONS ( ${${pkg}_DEFINITIONS} )
     ENDIF()
 ENDFOREACH()
 
+IF( USE_SDHCALCONTENT )
+    FIND_PACKAGE( SDHCALContent REQUIRED )
+    IF( SDHCALContent_FOUND )
+        INCLUDE_DIRECTORIES( SYSTEM ${SDHCALContent_INCLUDE_DIRS} )
+        LINK_LIBRARIES( ${SDHCALContent_LIBRARIES} )
+        ADD_DEFINITIONS ( ${SDHCALContent_DEFINITIONS} )
+        ADD_DEFINITIONS( "-DSDHCALCONTENT" )
```

```suggestion
        ADD_DEFINITIONS( "SDHCALCONTENT" )
```
Cmake will add the -D where necessary